app/src/main/java/org/sample/SampleRxActivity.kt

```diff
@@ -24,7 +24,7 @@ class SampleRxActivity : FluxRxActivity() {
 
         dummyStore.flowable()
             .subscribe {
-                demo_text.text = it.text
+                findViewById<TextView>(R.id.demo_text).text = it.text
```

Comment:
Gotta upgrade the example to ViewBinding xDD

Reply:
5f2e43c45d2de522e798fac62786ac27e393d960